c/parallel/test/test_segmented_reduce.cpp

```diff
@@ -170,7 +189,7 @@ extern "C" __device__ unsigned long long {0}(
 
   // a copy of offset iterator, so no need to define advance/dereference bodies, just reused those defined above
   iterator_t<SizeT, row_offset_iterator_state_t> end_offset_it = make_iterator<SizeT, row_offset_iterator_state_t>(
-    "", {advance_offset_method_name, ""}, {deref_offset_method_name, ""});
+    {offset_iterator_state_name, ""}, {advance_offset_method_name, ""}, {deref_offset_method_name, ""});
```

Comment:
Would just change `""` to something like `REUSE_SRC_DEFINITION`

Reply:
Renamed